api/util/mappers/dynamodb.py

```diff
@@ -77,10 +77,12 @@ def map_environment_api_key_to_environment_api_key_document(
 def map_engine_identity_to_identity_document(
     engine_identity: "IdentityModel",
 ) -> Document:
-    return {
+    response = {
         field_name: _map_value_to_document_value(value)
         for field_name, value in engine_identity
     }
+    response["composite_key"] = engine_identity.composite_key
+    return response
```

Comment:
I'm a little unsure why this had to change @khvn26 @zachaysan ?

Reply:
I believe it was an update to pydantic making computed properties no longer part of the generated document. @khvn26 had even opened an issue on their repo about it and the conclusion was that it was not going to get fixed at least until the first major upgrade.